modules/widgets/synergywholesale_servercost.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+use WHMCS\Database\Capsule as DB;
+use WHMCS\Module\AbstractWidget;
+
+class CostWidgetAPI
+{
+    protected $resellerID;
+    protected $apiKey;
+    protected $soap;
+
+    public function __construct($resellerID, $apiKey)
+    {
+        $this->soap = new SoapClient(null, array("location" => "https://api.synergywholesale.com", "uri" => "", 'trace' => true));
+        $this->apiKey = $apiKey;
+        $this->resellerID = $resellerID;
+    }
+
+    public function request($command, $params = [])
+    {
+        $params['resellerID'] = $this->resellerID;
+        $params['apiKey'] = $this->apiKey;
+
+        try {
+            $response = $this->soap->$command($params);
+        } catch (Exception $e) {
+            // Do stuff?
+        }
+
+        return $response;
+    }
+
+    public function balanceQuery()
+    {
+        return $this->request('balanceQuery');
+    }
+
+    public function listHosting()
+    {
+        return $this->request('listHosting');
+    }
+
+    public function hostingListPackages()
+    {
+        return $this->request('hostingListPackages');
+    }
+}
+
+class CostWidget extends AbstractWidget
+{
+    protected $title = 'Synergy Wholesale Server Cost';
+    protected $description = '';
+    protected $weight = 150;
+    protected $columns = 1;
+    protected $cache = false;
+    protected $cacheExpiry = 120;
+    protected $requiredPermission = '';
+
+    public function getData()
+    {
+        $query = DB::table('tblproducts')
+            ->where('servertype', 'synergywholesale_hosting')
+            ->first();
+
+        $result['api_key'] = $query->configoption1;
+        $result['reseller_id'] = $query->configoption2;
+
+        return $result;
+    }
+
+    public function generateOutput($result)
+    {
+        // Get if button is clicked
+        $syncCost = App::getFromRequest('sync_cost');
+        $content = "";
+
+        // Find current SWS Server
+        $query = DB::table('tblservers')
+            ->where('name', 'Synergy Wholesale')
+            ->first();
+
+        // Check if SWS server is found
+        if (!empty($query)) {
```

Comment:
Can we please look at bailing out early rather than nesting these if statements like this?

Such as:
```php
if (empty($query)) {
    return $defaultContent;
}
```